packages/key-management/src/types.ts

```diff
@@ -69,6 +75,11 @@ export interface AccountAddressDerivationPath {
   index: number;
 }
 
+export interface SharedWalletAddressDerivationPath {
```

Comment:
Do we still need this?

Reply:
Leaving this was an oversight. Removed on c672ad7